### PR TITLE
ghostty 1.0.0 (new cask)

### PR DESCRIPTION
**Pull request by Uzaaft**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by p-linnane** [comment]
Please run `brew style --fix` and rebase against master so we can properly review this. Thanks.

**Comment by Uzaaft** [comment]
Its a draft PR after all.

File: Casks/g/ghostty.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+cask "ghostty" do
+  version "1.0.0"
+  sha256 "091f7a2b3f4160a16d7d52b2822124bb9d5714993815f62a7d70027984372652"
+
+  url "https://release.files.ghostty.org/#{version}/Ghostty.dmg"
+  name "Ghostty"
+  desc "Fast, native, feature-rich terminal emulator pushing modern features"
```
Comment (p-linnane) [inline Casks/g/ghostty.rb:7]:
```suggestion
  desc "Terminal emulator that uses platform-native UI and GPU acceleration"
```
See https://docs.brew.sh/Cask-Cookbook#stanza-desc.

File: Casks/g/ghostty.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+         target: "#{HOMEBREW_PREFIX}/share/fish/vendor_completions.d/ghostty.fish"
+  binary "#{appdir}/Ghostty.app/Contents/Resources/terminfo/67/ghostty",
+         target: "#{ENV.fetch("TERMINFO", "~/.terminfo")}/67/ghostty"
+  binary "#{appdir}/Ghostty.app/Contents/Resources/terminfo/78/xterm-ghostty",
+         target: "#{ENV.fetch("TERMINFO", "~/.terminfo")}/78/xterm-ghostty"
+  manpage "#{appdir}/Ghostty.app/Contents/Resources/man/man1/ghostty.1"
+  manpage "#{appdir}/Ghostty.app/Contents/Resources/man/man5/ghostty.5"
+
+  zap trash: [
+    "/Applications/Ghostty.app/Contents/Resources/terminfo/67/ghostty",
```
Comment (p-linnane) [inline Casks/g/ghostty.rb:25]:
```suggestion
```
This will be removed when the .app is removed.

**Review by p-linnane — APPROVED** [review]
Thank you!

**Comment by Uzaaft** [comment]
Sneaked in one last change that i didnt get @p-linnane. Should be ready for review now. :)

**Review by p-linnane — APPROVED** [review]

**Review by SMillerDev — APPROVED** [review]

**Review by krehel — APPROVED** [review]

**Comment by krehel** [comment]
Thanks so much @Uzaaft for the submission!

**Pull request merged**